draft-ietf-quic-recovery.md

```diff
@@ -577,15 +577,22 @@ Pseudocode for OnPacketSent follows:
 
 ### On Ack Receipt
 
-When an ack is received, it may acknowledge 0 or more packets.
+When an acknowledgment is received, it may acknowledge 0 or more packets.
+
+The first acknowledgment for the largest acknowledged packet is used to update
+the estimate of minimum and smoothed RTT.  However, if the largest acknowledged
+packet would not have caused an acknowledgment to be sent, RTT estimates are not
+updated, because the acknowledgment could be significantly delayed.
 
 Pseudocode for OnAckReceived and UpdateRtt follow:
 
 ~~~
    OnAckReceived(ack):
      largest_acked_packet = ack.largest_acked
-     // If the largest acked is newly acked, update the RTT.
-     if (sent_packets[ack.largest_acked]):
+     // If the largest acked is newly acked and is a packet
+     // that would normally cause an ack, update the RTT.
+     if (sent_packets[ack.largest_acked] and
+         not ack_only(sent_packets[ack.largest_acked])):
```

Comment:
We use "ackable" as a term in this doc to refer to a packet that triggers an ack. I just realized that it's not yet defined in the doc, but I would use that term ("and is_ackable(sent_packets[ack.largest_acked])"). I'm happy to write up a definition for "ackable" in a PR.

Alternatively, you can leave this as you have it, and I can change it when I actually define the term. Up to you.